apps/web/src/ee/clerk/pages/SignUpPage.tsx

```diff
@@ -0,0 +1,15 @@
+import { SignUp } from '@clerk/clerk-react';
+import AuthLayout from '../../../components/layout/components/AuthLayout';
+import { ROUTES } from '../../../constants/routes';
+
+export default function SignUpPage() {
+  return (
+    <AuthLayout title="Sign Up" description="Hello and welcome! Sign up to the best notifications platform ever">
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is this a temporary description? We may want marketing's input here‏

Reply:
Yes, all descriptions and styles on the Clerk components are just temporary and will change.